.github/workflows/release-infra-test.yaml

```diff
@@ -0,0 +1,98 @@
+name: Test the ARM64 runner infrastructure
+on:
+  pull_request: # TODO: make absolutely sure to remove this before merging
```

Comment:
I'm adding a review comment and request changes so that we don't forget to do so.